src/NuGet.Core/NuGet.Packaging/Signing/SignManifest.cs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using NuGet.Versioning;
+
+namespace NuGet.Packaging.Signing
+{
+    /// <summary>
+    /// Represents the XML manifest containing a set of files and hashes that can be signed.
+    /// </summary>
+    public class SignManifest
```

Comment:
`sealed`

Reply:
I'm curious: **why sealing?** (here and elsewhere in this PR)
Sealing is explicit and tells callers they *cannot and may not* extend this class. 
Not sealing means *we don't care really* whether one extends or not.

Even though I'm not immediately seeing a use case for extending these, we're not really being open for extensibility by sealing everything by default. 
So unless there's a real explicit reason to do so, I would not mark all of these as `sealed`. Thoughts?